__tests__/app.test.js

```diff
@@ -36,4 +36,42 @@ describe("GET /api" ,() => {
     expect(res.body.endpoints).toEqual(JSON.parse(jsonFileData))
     })
     })
+})
+
+describe("GET /api/articles/:article_id", () => {
+    test("status: 200, responds with an article of a specific ID from the GET request", () => {
+        return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then((res) => {
+            expect(res.body).toMatchObject( {
+                article: {
+                    author: expect.any(String),
+                    title: expect.any(String),
+                    article_id: expect.any(Number),
+                    body: expect.any(String),
+                    topic: expect.any(String),
+                    created_at: expect.any(String),
+                    votes: expect.any(Number),
+                    article_img_url: expect.any(String)
+                }
+            })
+        })
+    })
+    test("status: 404, responds with 'The request path does not exist' if the endpoint does not exist", () => {
```

Comment:
You only need to test this one time in your whole test file. As long as you have a test somewhere that checks the response when a non-existent endpoint is requested then that is enough, so this may not be needed if you have checked this elsewhere?